dpnp/dpnp_iface_histograms.py

```diff
@@ -752,3 +775,310 @@ def histogram_bin_edges(a, bins=10, range=None, weights=None):
     a, weights, usm_type = _ravel_check_a_and_weights(a, weights)
     bin_edges, _ = _get_bin_edges(a, bins, range, usm_type)
     return bin_edges
+
+
+def _histdd_validate_bins(bins):
+    for i, b in enumerate(bins):
+        if numpy.ndim(b) == 0:
+            if b < 1:
+                raise ValueError(
+                    f"'bins[{i}' must be positive, when an integer"
+                )
+        elif numpy.ndim(b) == 1:
+            # will check for monotonicity later
+            pass
+        else:
+            raise ValueError(
```

Comment:
Could you please add a test to cover that?

Reply:
added